run-tests.sh

```diff
@@ -161,13 +165,20 @@ test_galaxy_internal() {
 }
 
 test_doc() {
-	mvn clean site $@
+  mvn clean site $@
+	exit_code=$?
+	return $exit_code
+}
+
+test_open_api() {
+  mvn clean verify -B -Dspring.profiles.active=dev,swagger -DskipTests=true -Dliquibase.update.database.schema=true -Djdbc.url=$JDBC_URL -Djdbc.pool.maxWait=$DB_MAX_WAIT_MILLIS
+	test -f $OPEN_API_FILE
```

Comment:
Got some weird formatting going on in here.  Otherwise I'm good with the changes here to add the extra testing step.

Reply:
Please see 83e502aa59dcdf8ba1df65225b57a422612ef126.